router/smoke/openshift_2/test_openshift_01_scale_up.py

```diff
@@ -0,0 +1,56 @@
+import time
+
+from messaging_components.routers import Dispatch
+from messaging_components.routers.dispatch.management import RouterQuery
+from typing import Tuple
+
+from router.smoke.openshift_2.ocp_util import OpenShiftUtil
+
+MESH_SIZE = 3
+
```

Comment:
I have also separated the tests that were inside test_openshift_template.py into test_openshift_01_scale_up.py and test_openshift_03_scale_down.py so we can observe scaling happening and messages being exchanged while router is running with a formed mesh.

I believe it is also good to use "01", "02", ... notation in our tests for a given topology, in order to guarantee things happen in the correct order.

Reply:
It seems to me that depends on how we want to write a test. If we want to write one "common" compatible test suite where doesn't matter on how much clusters you use then dynamic with right usage is enough.